docs/about_GettingStarted.md

```diff
@@ -0,0 +1,167 @@
+
+```ps
+New-PolarisRoute -Method POST -Path "/" -ScriptBlock {
+    $Respond.Send('Received POST request')
+}
+```
+
+New PUT route to the /Process route:
+
+```ps
+New-PolarisRoute -Method PUT -Path "/Process" -ScriptBlock {
+    $Response.Send('Received a PUT request at the /Process route')
+}
+```
+
+New DELETE route at the /Process route:
+
+```ps
+New-PolarisRoute -Method DELETE -Path "/Process" -ScriptBlock {
+    $Response.Send('Received a DELETE request at the /Process route')
+}
+```
+
+See about_Routing for more details on advanced routing
+
+## $Request and$Response
+
+Polaris will create two automatic variables for you that are available inside the scriptblock for each route.
+
+\$Request represents information regarding the incoming request from the client and will have details like the URL requested, the body of the request, the query string, the parameters, etc.
+
+\$Response represents the response that will be sent to the client. You can set response headers, send strings or streams back to the client.
+
+## Static Files
+
+To serve static files for your site such as javascript, images, css and other assets you can use a static file route. Polaris takes the contents of an entire folder and moves them to a temporary PSDrive to prevent directory traversal and then serves any files in that folder automatically detecting the MIME type for the file to send with the response.
+
+Let's say you have simple static website that you want to serve using Polaris. It's made up of the following files in a folder on the C drive called MyAwesomeSite. The files are as follows:
+
+1. **Index.html** - The home page
+2. **Scripts.js** - A few handy javascript functions
+3. **MyStyles.css** - Some css to make my site stand out
+4. **favicon.ico** - Because every good site nees an icon
+
+I can serve all of these files automatically using the following command:
+
+```ps
+New-PolarisStaticRoute -RoutePath "/" -FolderPath "C:\MyAwesomeSite"
+```
+
+Now I can just call `Start-Polaris` and the following requests will work from a browser:
+
+```
+http://localhost:8080/index.html
+http://localhost:8080/scripts.js
+http://localhost:8080/mystyles.css
+http://localhost:8080/favicon.ico
+```
+
+Polaris also has a built-in directory browser you can enable that can be used to set up a simple file server:
+
+```ps
+New-PolarisStaticRoute -RoutePath "/" -FolderPath "C:\MyFolderOfFiles" -EnableDirectoryBrowser $True
+```
+
+## Simple APIs
+
+APIs are all the rage today and working with them from a browser means you are going to want JSON. PowerShell provides a serious help to you natively with the `ConvertTo-Json` command and Polaris provides a nice helper method to specify that the string you are sending to the client is JSON.
+
+Let's create a quick API for showing information about running processes:
+
+```ps
+New-PolarisRoute -Method GET -ScriptBlock {
+    $ProcessInfo = Get-Process | Select ProcessName,Id,CPU | ConvertTo-Json | Out-String
```

Comment:
```suggestion
    $ProcessInfo = Get-Process | Select ProcessName,Id,CPU | ConvertTo-Json
```

The `Out-String` should be unnecessary because `ConvertTo-Json` already returns a string.